tools/eventlist/pkg/output/output.go

```diff
@@ -72,6 +73,7 @@ type EventRecord struct {
 	Component     string  `json:"component" xml:"component"`
 	EventProperty string  `json:"eventProperty" xml:"eventProperty"`
 	Value         string  `json:"value" xml:"value"`
+	Level         string  `json:"level" xml:"level"`
```

Comment:
I've exposed `level` only in xml and json output. Do we also want another column in txt format to display level?

Reply:
@jkrech Do we want to display `level` also in txt output or just in xml and json?

Reply:
In my view this is a matter of consistency and intended use cases.
As far as I am aware, the level filtering can be configured in the embedded application already.
Also filtering by level could be performed based on the scvd information associated with the eventID, such that we could expand the eventlist utility adding the level filtering to the actual processing, rather than adding it as an additional field to the output.

@bgn42, could you please check the output format of uVision's eventrecorder whether it includes level information or not?

Reply:
@bgn42 Any news about the `level` in uVision's output?

Reply:
For consistency I've removed the `level` from output. This PR now only allows to filter events by level.